### PR TITLE
USWDS-Site - Migration guide: Add note about autoprefixer

### LINKED ISSUE #2296: USWDS-Site: Add note about Autoprefixer to migration guide
## Summary
Add a note about Autoprefixer to the [Migration guide page](https://designsystem.digital.gov/documentation/migration/#3-update-your-sass-compiler-settings-and-recompile-css). This is important for users who are exclusively using the Migration guide to set up their builds. 

It should be similar to the note we have in the [getting started for devs guide](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/#loading-uswds-in-your-sass-entry-point): 
> Note: USWDS relies on [Autoprefixer](https://github.com/postcss/autoprefixer) to make its compiled CSS browser-compatible. If your compiler does not already use Autoprefixer, we strongly encourage that you add it to your project.

More details in the related uswds-public Slack comment: https://gsa-tts.slack.com/archives/C3F14AHSQ/p1696437418207649?thread_ts=1696434410.341549&cid=C3F14AHSQ

### PR DESCRIPTION
**Pull request by mahoneycm**
# Summary

Moved `Autoprefixer` note to an `include` and added the note to the Migrating to 3.0 page. 

> ~**Warning**~
> ~Ignored two snyk vulnerabilities causing test failures. This can be resolved by #2322~

_Warning resolved and can be ignored. Leaving for historical context._

## Related issue

Closes #2296

## Preview link

[Migrating to 3.0 page →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/#3-update-your-sass-compiler-settings-and-recompile-css-2)
[Phase 2: Compile →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/getting-started/developers/phase-two-compile//)

[Changelog →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/#latest-updates)

> **Note**
> Must update changelog date before merge

## Problem statement

Issue arose from a  [discussion](https://gsa-tts.slack.com/archives/C3F14AHSQ/p1696437418207649?thread_ts=1696434410.341549&cid=C3F14AHSQ) in the public slack channel. Some users may be using the migration guide exclusively to set up their builds. Without this important piece of information, these users may run into issues with their CSS.

## Solution

Use note about Autoprefixer from Phase 2: Compile page in the Sass compiler instructions.

## Testing and review

1. Inspect new `autoprefixer-note.html` does have gramattical errors.
2. Visit the [Phase 2: Compile](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/getting-started-for-developers/) page
3. Confirm Autoprefixer note matches what's on [develop](https://designsystem.digital.gov/documentation/getting-started/developers/phase-two-compile/).
5. Visit the [Migrate to 3.0](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-migration-autoprefixer/documentation/migration/) page
6. Inspect new note re: Autoprefixer
7. Confirm placement is appropriate

### CODE REVIEW

**Comment by mahoneycm** [comment]
@amyleadem Good thinking! Changed pushed up and conflicts resolved!

**Review by amyleadem — APPROVED** [review]
Looks good to me!
- [x] Confirmed that the site note on the compile page is unchanged
- [x] Confirmed that the migration guide now has a note about autoprefixer
- [x] Confirmed the note on the migration page is in a logical location
- [x] Confirmed the note on the migration page has content that makes sense in context
- [x] Confirmed the changelog is accurate and makes sense

**Review by mejiaj — CHANGES_REQUESTED** [review]
LGTM, changelog date and preview link for `phase-two` needs to be updated. Good to merge after.

**Comment by mahoneycm** [comment]
@mejiaj I've updated the changelog date and `phase-two` link. Pa11y tests are failing in the pipeline but working passing!

File: _data/changelogs/docs-migration-v3.yml [hunk 1]
```diff
@@ -2,6 +2,11 @@ title: Migrating to 3.0
 type: documentation
 changelogURL:
 items:
+  - date: 2023-11-29
```
Comment (mahoneycm) [inline _data/changelogs/docs-migration-v3.yml:5]: > [!IMPORTANT]
> Neds updated changelog date on approval

Comment (mahoneycm) [reply _data/changelogs/docs-migration-v3.yml]: Resolved in [a41ec1a](https://github.com/uswds/uswds-site/pull/2319/commits/a41ec1a53e6079f994cf24ef9a89dc5febc87253)

**Review by mejiaj — APPROVED** [review]

**Comment by mahoneycm** [comment]
@thisisdano quick win PR ready for your review!

File: _data/changelogs/docs-migration-v3.yml [hunk 2]
```diff
@@ -2,6 +2,11 @@ title: Migrating to 3.0
 type: documentation
 changelogURL:
 items:
+  - date: 2024-04-29
```
Comment (thisisdano) [inline _data/changelogs/docs-migration-v3.yml:5]:
```suggestion
  - date: 2024-06-07
```

**Pull request merged**